packages/client/src/client/App/Trades/TradesState/tableTrades.ts

```diff
@@ -357,33 +359,23 @@ export const [creditTradeRowHighlight$, setCreditTradeRowHighlight] =
 /**
  * Emit tradeId of new trades after the initial load
  */
-const newTradeId$ = merge(trades$, creditTrades$).pipe(
-  scan(
-    (acc, trades) => {
-      return {
-        stateOfWorld: acc.stateOfWorld && acc.trades.length === 0,
-        trades: trades,
-        skip: acc.trades.length === trades.length,
-      }
-    },
-    { stateOfWorld: true, trades: [], skip: false } as {
-      stateOfWorld: boolean
-      trades: TradeType[]
-      skip: boolean
-    },
-  ),
-  filter(
-    ({ stateOfWorld, trades, skip }) =>
-      !stateOfWorld && trades.length > 0 && !skip,
+const newTradeId$ = tradesStream$.pipe(
```

Comment:
simplified new trade ID capture streams, going back to the more fundamental source streams, which retain the "state of the world" flag for FX and the appropriate quoteAccepted update for Credit